build.py

```diff
@@ -18,6 +18,8 @@
 summary = "PyBuilder plugin to handle AWS functionality"
 authors = [Author('Valentin Haenel', 'valentin@haenel.co'),
            Author('Stefan Neben',    'stefan.neben@gmail.com'),
+           Author('Carsten Rose',    'modebm@gmail.com'),
+           Author('Schlomo Schapiro','schlomo.schapiro@gmail.com')
```

Comment:
Please place a comma at the end of this line to encourage a defensive programming style